dfx.json

```diff
@@ -0,0 +1,46 @@
+{
+  "canisters": {
+    "signer": {
+      "frontend": {
+        "entrypoint": "examples/signer/dist/index.html"
+      },
+      "source": [
+        "examples/signer/dist/"
+      ],
+      "type": "assets"
+    },
+    "dapp": {
+      "frontend": {
+        "entrypoint": "examples/dapp/dist/index.html"
+      },
+      "source": [
+        "examples/dapp/dist/"
+      ],
+      "type": "assets"
+    }
+  },
+  "defaults": {
+    "build": {
+      "packtool": ""
+    }
+  },
+  "networks": {
+    "local": {
+      "bind": "127.0.0.1:8000",
+      "type": "ephemeral"
+    },
+    "ic": {
```

Comment:
here `prod` in the case I've mentioned above

Reply:
The akward issue is here, when I try to use prod or any other name for mainnet. The separate ticket will be created to handle this.